### DISCUSSION TITLE
L'assemblage avec MetaSpades ne me donnent pas les fichiers résultats (contigs.fasta et scaffolds.fasta)

### DISCUSSION THREAD

**Discussion by fatimazahraabani** [ablab/spades#1144] [General]
Bonjour,

J'ai fait un assemblage avec metaspades sur un ensemble de reads paired-end mais à chaque fois le dossier de résultats ne contient pas les fichiers contigs.fasta ni scaffolds.fasta, pourtant j'en avait besoin pour passer à une analyse de qualité avec quast. je sais pas où est l'erreur
![Capture d’écran du 2023-05-26 10-09-12](https://github.com/ablab/spades/assets/119899025/902d80f1-71f5-4135-b134-5c69bbc4d7ce)

**Comment by asl** [comment]
Возможно, Вы быстрее получите ответ, задав вопрос на более привычном языке.
